### DISCUSSION TITLE
Project Proposals and Ideas for GSoC 2023

### DISCUSSION THREAD

**Discussion by peb-peb** [google/gcp_scanner#65] [General]
This is the discussion where we could discuss ideas for GSoC 2023 or review project proposals for GSoC 2023.

**Comment by tausiq2003** [comment]
Hey there I made something to add functions into `main` function under project idea of code refactoring.

**Reply by tausiq2003** [reply]
should I represent it on proposal?

**Reply by peb-peb** [reply]
yeah, since the proposal requires you to mention your open-source proposals...
